tool/lib/can_actions.py

```diff
@@ -151,6 +156,7 @@ def bruteforce_arbitration_id(self, data, callback, min_id, max_id,
             extended = False
             if arb_id > ARBITRATION_ID_MAX:
                 extended = True
+            data+=[0]*(8-len(data))
```

Comment:
Why is the behavior of this method changed for this PR? This seems to make it impossible to send non-padded data through an arbitration ID bruteforce, which reduces its extensibility. It should rather be the responsibility of the calling function to provide data on the exact format to use for bruteforcing.